src/main/java/com/devotedmc/ExilePearl/ExilePearlPlugin.java

```diff
@@ -61,4 +63,8 @@ public boolean onCommand(CommandSender sender, Command cmd, String alias, String
 	public List<String> onTabComplete(CommandSender sender, Command cmd, String alias, String[] args) {
 		return core.onTabComplete(sender, cmd, alias, args);
 	}
+
+	public static ExilePearlPlugin getInstance() {
```

Comment:
Do we need this? Exile pearl usually has a way to avoid using the plugin instance and this class isnt the main plugin class

Reply:
I'd recommend asking Ori. He did that.